packages/vite/src/node/plugin.ts

```diff
@@ -291,7 +292,10 @@ export interface Plugin<A = any> extends RollupPlugin<A> {
    * `{ order: 'pre', handler: hook }`
    */
   transformIndexHtml?: IndexHtmlTransform
-
+  /**
+   * Build Environments
```

Comment:
```suggestion
   * Build Environments
   *
   * @experimental
```
I guess it should be marked as experimental.